docs/model_server_rest_api.md

```diff
@@ -180,4 +183,128 @@ A request in [column format](https://www.tensorflow.org/tfx/serving/api_rest#spe
   "outputs": <value>|<(nested)list>|<object>
 }
 ```
-Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
\ No newline at end of file
+Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
+
+## Config Reload API <a name="config-reload"></a>
+* Description
+
+Sends requests via RESTful API to trigger config reloading and gets models and [DAGs](./dag_scheduler.md) statuses as a response.
+
+This endpoint can be used with disabled automatic config reload to ensure changes in configuration are applied in a specific time and also to get confirmation about reload operation status. Typically this option is to be used when OVMS is started with a parameter --file_system_poll_wait_seconds 0.
+Reload operation does not pass new configuration to OVMS server. The configuration file changes needs to be applied by the OVMS administrator. The REST API call just initiate applying the configuration file which is already present.
+
+Flow after receiving request:
+1) If config file was changed - reload config.
+2) If any model version directory was changed or new version was added - reload this model.
+3) If any model that is part of a DAG was changed or new version dir was added - reload this pipeline.
+4) In case there are no errors in the reload operation, the response includes the status of all models and DAGs. If any of those first 3 steps causes reload - return code is 201, otherwise 200.
```

Comment:
maybe it would be clearer to use a table with the returned status codes:
200 - config applied, no changes were needed
201 - changed applied successfully
412 - error in applying some or all changes.